.prh.yaml

```diff
@@ -15,6 +15,7 @@
 #
 version: 1
 # index: ann
+# index: burstable
```

Comment:
it would be better to add Guaranteed and BestEffort and QoS

Reply:
@kpango updated